core/include/dt-bindings/clock/at91.h

```diff
@@ -27,6 +27,7 @@
 #define PMC_PLLBCK		8
 #define PMC_AUDIOPLLCK		9
 #define PMC_MCK_PRES		10
+#define PMC_CORE_CLK_COUNT	11
```

Comment:
What about SAMA7G5 variant with specific belo clocks ?

Reply:
You are right, this should actually be named `PMC_SAMA5D2_CORE_CLK_COUNT`since this is specific to sama5d2 platform.